### DISCUSSION TITLE
Pull down resistors

### DISCUSSION THREAD

**Discussion by gregcope** [boatybits/boatyFridgePy#4] [General]
HI,

What pull down resistors would you add to the N-Fet pins?

**Comment by boatybits** [comment]
Hi Greg,not sure it matters that much, 5k or something like that would probably be ok. I'm wondering now if a normal transistor might be better for control. Had a play with a fet and an adjustable power supply but a few millivolts went for off to full revs but it worked. Though the v1 circuitboard works, just made a mistake of putting the Q1 FET with no resistance in the circuit onto the LED pin so can't use it and can run the compressor with at the lowest revs.  Maybe with that on another pin it would be fine.
